css/components/card.css

```diff
@@ -0,0 +1,120 @@
+/* === css for cards === */
+
+.card{
+    display: flex;
+    flex-direction: column;
+    border: .1rem solid rgb(128, 122, 122);
```

Comment:
you should use colour variable

Reply:
actually I have already made my color Palette, so just to give border a color , I have use RGBA declaration.